src/ontology/mondo-edit.obo

```diff
@@ -166528,7 +166530,7 @@ xref: UMLS:C0043459 {source="MONDO:subClassOf", source="OMIM:214100", source="NC
 xref: UMLS:C1865803 {source="OMIM:214100", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C1865804 {source="OMIM:214100", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0019234 {source="MONDO:0008953/inferred", source="MONDO:Redundant", source="MONDOLEX:0008953/inferred", source="OMIM:214100", source="indirect"} ! peroxisome biogenesis disorder
-is_a: MONDO:0019609 {source="DC-OMIM:214100", source="MONDOLEX:0008953", source="ORDO-Gene"} ! Zellweger syndrome
+is_a: MONDO:0100259 {source="DC-OMIM:214100", source="MONDOLEX:0008953", source="ORDO-Gene", source="https://github.com/monarch-initiative/mondo/issues/2632", source="https://www.clinicalgenome.org/affiliation/40049/"} ! peroxisome biogenesis disorder due to PEX1 defect
```

Comment:
If the subclass axiom changed, shouldn't the sources be removed? Are these "sources" now really the sources of the subsumption? Also, what is `DC-OMIM:`?

Reply:
DC-OMIM is disease cluster OMIM. It was something we were using before Mondo was created. Chris has retained these IDs for historical purposes.